int8 actuatorNumber
int32 activationTime
